src/domains/nodo-secret/99_locals.tf

```diff
@@ -4,4 +4,6 @@ locals {
 
   monitor_action_group_slack_name = "SlackPagoPA"
   monitor_action_group_email_name = "PagoPA"
+
+  pipe_service_principel_application_id = "b7375314-3eff-4760-9952-84f96ee669cf"
```

Comment:
can you put a comment with the name of the SP, to simplify the search pleasee

Reply:
with application ID in Azure Portal search is more quickly then with name. use name of SP for search give more than one result